.github/workflows/backend-cd-gcp.yml

```diff
@@ -44,15 +44,26 @@ jobs:
 
           echo "DOCKER_IMAGE=$DOCKER_REGISTRY_HOST/$PROJECT_ID/titus/backend:$GITHUB_SHA" >> $GITHUB_ENV
 
-      - name: Build container
+      - name: Install Grype tool (vulnerability scan)
+        run: curl -sSfL https://raw.githubusercontent.com/anchore/grype/main/install.sh | sh -s -- -b /usr/local/bin
+
+      - name: Build Docker image
         working-directory: ./packages/titus-backend
-        run: |
-          docker build -t $DOCKER_IMAGE .
+        run: docker build -t $DOCKER_IMAGE .
+
+      - name: Scan image for vulnerabilities
+        run: /usr/local/bin/grype $DOCKER_IMAGE | tee /tmp/vuln-scan-titus-backend.txt
+
+      - name: Publish image scan results
+        uses: actions/upload-artifact@v2
+        with:
+          name: vuln-scan-titus-backend.txt
+          path: /tmp/vuln-scan-titus-backend.txt
+          retention-days: 15
```

Comment:
because installation, execution and publishing of the artifact is something we're doing over and over, why don't we extract that into a reusable github workflow? see https://docs.github.com/en/actions/learn-github-actions/reusing-workflows

Reply:
I'll have a look at it and see what can be done.

Reply:
let's do this also as a learning experience

Reply:
First working version of the reusable workflow for Grype: https://github.com/nearform/titus/actions/runs/1653415544

Reply:
Now I have to add this to the rest of workflows that build Docker images.

Reply:
Done! All the CD workflows have run successfully, except for the `storybook` one, which has an unrelated pending issue.

- Backend: https://github.com/nearform/titus/actions/runs/1653640186
- DB Manager: https://github.com/nearform/titus/actions/runs/1653640197
- Frontend: https://github.com/nearform/titus/actions/runs/1653640185